dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetVersionDifference.java

```diff
@@ -29,11 +31,13 @@
     private List<FileMetadata> addedFiles = new ArrayList<>();
     private List<FileMetadata> removedFiles = new ArrayList<>();
     private List<Tuple2<FileMetadata, FileMetadata>> changedFileMetadata = new ArrayList<>();
+    private List<Tuple2<FileTermsOfUse, FileTermsOfUse>> changedFileTerms = new ArrayList<>();
```

Comment:
In terms of tuples I absorbed the convention of naming it like `leftvalueAndRightvalue` so  I can see by the name which corresponds to the left and right. Here I feel like I can only assume that the left value are old terms and right one are new.

Reply:
That's why I'm also not a fan of Tuples larger than Tuple2 since it is becoming very hard to describe in the code what all the types mean.

Reply:
:+1: